autoscale/src/main/java/com/sequenceiq/periscope/monitor/event/ScalingEvent.java

```diff
@@ -13,9 +12,9 @@
 
     private List<String> decommissionNodeIds;
 
-    private transient Optional<Integer> scalingNodeCount = Optional.empty();
+    private transient Integer hostGroupNodeCount;
 
-    private transient Optional<Integer> hostGroupNodeCount = Optional.empty();
+    private transient Integer desiredHostGroupNodeCount;
```

Comment:
qualify the name with incremental or absolute to make it more clear?

Reply:
done